### DISCUSSION TITLE
How to pause simulated navigation?

### DISCUSSION THREAD

**Discussion by jsbien** [osmandapp/OsmAnd#14609] [Q&A]
Following the advice https://groups.google.com/g/osmand/c/DeGvZPeMwg0/m/amaDZf7jAQAJ I added "Pause / Resume navigation" to the "Quick Action" button.
When I tap "Pause" the buttom changes its name to "Resume" but the simulated movement continues.
This is version 4.2.4 on Android 9 (Neffos C9 Max).
How to solve or circumvent the problem?

**Comment by C6competition** [comment]
I have the same issue as @jsbien. Quick Menue Stop/Resume doesn't work in simulating navigation !!
It would be extremly helpfull during debugging to have this facility!
Version 4.2.6 and Android 11 Blackview 9900E

**Comment by vshcherb** [comment]
You can Dismiss Route.

**Comment by C6competition** [comment]
With Dismiss Route you quit simulation of the route completely.
If you want to continue simulation you have to start again at the beginning!
What I think will be helpfull, is to pause the simulation, make a screenshot, inspect the situation etc.
and then continue simulating!

**Reply by jsbien** [reply]
I support this.

**Reply by vshcherb** [reply]
Looks like I answered to the question how ti "Quit simulation", I didn't try it but is Pause navigation works which you can see in notification message on the notification bar.

**Comment by C6competition** [comment]
That's correct!
Start/Stop navigation on Quick Menue works for simulation as well!
Pause/Resume navigation on the Quick Menue does NOT work for simulation!
